.credo.exs

```diff
@@ -130,11 +130,7 @@
 
         # Deprecated checks (these will be deleted after a grace period)
         #
-        {Credo.Check.Readability.Specs, false},
```

Comment:
I think we should keep these checks, just ignore for some files or fix the occurances